packages/components/InputPanel.tsx

```diff
@@ -0,0 +1,60 @@
+import { Flex, Grid, GridItem } from '@chakra-ui/react';
+
+import BaseButton from './BaseButton';
+
+const generateNumberList = (max = 10) => {
+  return Array(max)
+    .fill(null)
+    .map((_, i) => (i + 1) % max);
+};
+
+const numericButtons = generateNumberList().map((btnText) => {
+  return (
+    <GridItem
+      key={btnText}
+      w="100%"
+      h="100%"
+      colStart={btnText === 0 ? 2 : undefined}
+    >
+      <BaseButton
+        onClick={() => {}}
+        text={btnText.toString()}
+        variant="number"
+      />
+    </GridItem>
+  );
+});
```

Comment:
Exemplo do que quis dizer  acima. Ainda falta tipar as coisas, mas o children seria string e o onClick uma function.

```suggestion
const NumericButton = (props) => {
  const { children, onClick } = props
  const colStart = btnText === 0 ? 2 : undefined
  return (
    <GridItem w="100%" h="100%" colStart={colStart} >
      <BaseButton
        onClick={onClick}
        text={children}
        variant="number"
      />
    </GridItem>
  );
};
```